package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "pc-nrfconnect-shared",
-    "version": "4.29.0",
+    "version": "4.29.1",
```

Comment:
It seems like 4.29.0 was not yet released, so there is no need to bump the version.